python/tvm/topi/arm_cpu/mprofile/dsp/micro_kernel/common.py

```diff
@@ -29,3 +29,18 @@
 #include <tvm/runtime/crt/error_codes.h>
 
 """
+
+MICRO_WORD_LENGTH = 32
```

Comment:
Added the `BITS` unit. I'd argue against having it as a parameter, since this is located inside the `mprofile` folder, where word length will always be 32.